### ISSUE TITLE
Windows build failing after #204

### ISSUE THREAD

**Issue by NathanielMcVicar** [onnx/onnx-mlir#235]
PR #204 removed outputBaseName from the signature of genConstPackObj() in MainUtils.cpp, but this variable is used in an #ifdef path only reached on Windows. I think this may have gone undetected because of #234.

**Comment by caoimhinuibrian** [comment]
@gongsu832 see my comment in #234

**Comment by NathanielMcVicar** [comment]
@caoimhinuibrian Should we close this as a duplicate of 234? I wasn't sure about the project policy, but it seems like all the discussion is happening there. Thanks!

**Comment by caoimhinuibrian** [comment]
@NathanielMcVicar I'm not sure that we have an explicit policy, but closing this is probably fine.

**Comment by NathanielMcVicar** [comment]
Duplicate of #234
